tests/test_distribution.py

```diff
@@ -0,0 +1,126 @@
+# SPDX-Litense-Identifier:  MPL-2.0
+import unittest
+import numpy as np
+from rdflib import Graph, URIRef
+from rdf_utils.constraints import check_shacl_constraints
+from rdf_utils.models.distribution import (
+    DistributionModel,
+    SampledQuantityModel,
+    sample_from_distrib,
+)
+from rdf_utils.resolver import install_resolver
+from rdf_utils.uri import URL_MM_DISTRIB_JSON, URL_MM_DISTRIB_SHACL, URL_SECORO_M
+
+# random distribution params
+NUM_SAMPLE = 20
+DIM = 7
+RAND_NUMS = np.round(np.random.rand(DIM) * 100 - 50, decimals=3)  # numbers in range [-50, 50)
+RAND_RANGE = np.random.randint(10, 20, size=DIM)
+RAND_SQUARE_MAT = np.random.uniform(-50, 50, size=(DIM, DIM))
+RAND_COV = np.round(
+    np.dot(np.transpose(RAND_SQUARE_MAT), RAND_SQUARE_MAT), decimals=3
+)  # covariance of shape (DIM, DIM)
+
+
+def get_matrix_string(matrix):
+    row_strs = []
+    for row in matrix:
+        row_strs.append(f"[{", ".join(map(str, row))}]")
```

Comment:
it is throwing syntax error while running test cases. the below should work. 

```python
row_strs.append(f"[{', '.join(map(str, row))}]")
```

Reply:
resolved in minhnh/rdf-utils@63ba753f2162b688273d1eb3709155e2ce62a399